docs/en/ingest-management/fips-ingest.asciidoc

```diff
@@ -0,0 +1,119 @@
+[role="xpack"]
+[[fips-ingest]]
+= FIPS mode for Ingest tools
+
+preview::[]
+
+{agent}, {fleet}, {filebeat}, {metricbeat}, and APM Server binaries are built and are configured to use FIPS 140-2 compliant cryptography.
+Generally speaking FIPS 140-2 requirements can be summarized as:
+
+- linking against a FIPS certified cryptographic library
+- using only FIPS approved cryptographic functions
+- ensuring that the configuration of the component is FIPS 140-2 compliant.
+
+[discrete]
+[[fips-binaries]]
+== FIPS-compatible binaries and configuration 
+
+FIPS compatible binaries for {agent}, {fleet}, {filebeat}, {metricbeat}, and APM Server are available for link:https://www.elastic.co/downloads[download].
+Look for the `Linux 64-bit (FIPS)` or `Linux aarch64 (FIPS)` platform option on the product download pages for {agent} and {fleet}, {filebeat}, and {metricbeat}.
+Look for the `Linux x86_64 (FIPS)` or `Linux aarch64 (FIPS)` platform option on the APM Server download page.
+
+IMPORTANT: The default configurations provided in the binaries are FIPS compatible. Be sure to check and understand the implications of changing default configurations. 
+
+[discrete]
+[[ingest-limitations-all]]
+== Limitations 
+
+[discrete]
+[[ingest-limitations-tls]]
+=== TLS 
+
+Only FIPS 140-2 compliant TLS protocols, ciphers, and curve types are allowed to be used:
+
+* The supported TLS versions are `TLS v1.2` and `TLS v1.3`. 
+* The supported cipher suites are:
+
+** `TLS v1.2`: `ECDHE-RSA-AES-128-GCM-SHA256`, `ECDHE-RSA-AES-256-GCM-SHA384`, `ECDHE-ECDSA-AES-128-GCM-SHA256`, `ECDHE-ECDSA-AES-256-GCM-SHA384`
+** `TLS v1.3`: `TLS-AES-128-GCM-SHA256`, `TLS-AES-256-GCM-SHA384`
+
+* The supported curve types are `P-256`, `P-384` and `P-521`.
+
+Support for encrypted private keys is not available, as the cryptographic modules used for decrypting password protected keys are not FIPS validated. If an output or any other component with an SSL key that is password protected is configured, the components will fail to load the key. When running in FIPS mode, you must provide non-encrypted keys.
+Be sure to enforce security in your FIPS environments through other means, such as strict file permissions and access controls on the key file itself, for example. 
```

Comment:
Same thing here. Could this be presented as a list of recommendations / considerations?